src/phantom/_utils/types.py

```diff
@@ -1,6 +1,10 @@
+import sys
 from typing import TypeVar
 
-from numerary.types import CachingProtocolMeta  # type: ignore[attr-defined]
+if sys.version_info > (3, 7):
```

Comment:
```suggestion
if sys.version_info >= (3, 8):
```

I'm guessing we're still failing because 3.7.15 > 3.7?

Reply:
Yep, just realised, haha